docs/bouwblokken/publieke-inkoop/index.md

```diff
@@ -1,77 +1,141 @@
+- [Maak het leveren van bewijs voor het voldoen aan de vereiste onderdeel van de beoordeling van een inschrijving](../../maatregelen/leveren_bewijs_onderdeel_beoordeling_inschrijving.md/index.html)
+- [Menselijke tussenkomst is een vast onderdeel in een projecptlan of een déchargedocument](../../maatregelen/menselijke_tussenkomst_projectplan_en_dchargedocument/index.html)
+- [Neem de vereiste op als een subgunningscriteria bij gunningscriteria beste prijs-kwaliteitverhouding](../../maatregelen/neem_vereiste_op_als_subgunningscriteria/index.html)
+- [Neem het kunnen uitvoeren van een audit over de vereiste op in contractvoorwaarden en de contractovereenkomst](../../maatregelen/uitvoeren_audit_voor_naleving_vereiste/index.html)
+- [Neem het kunnen uitvoeren van een audit over de vereiste op in contractvoorwaarden en de contractovereenkomst](../../maatregelen/uitvoeren_audit_voor_naleving_vereiste/index.html)
+- [Restrisico's met betrekking tot schending auteursrechten zijn inzichtelijk gemaakt](../../maatregelen/omgaan_restrisico's_aanbiede_onderdeel_beoordelingsmaatrix/index.html)
+- [Stel archiefbescheiden vast](../../maatregelen/stel_archiefbescheiden_vast/index.html)
+- [Stel vast of het gaat om een algoritme en/of AI-systeem en wat de bijbehorende risicoclassificatie is om te bepalen welke vereisten hierop van toepassing zijn.](../../maatregelen/vaststellen_typen_algoritme_of_AI-systeem_en_risicoclassificatie/index.html)
+- [Vastellen niveau van benodigde training voor gebruik algoritmen en AI-systemen](../../maatregelen/vaststellen_passend_trainingsniveau_door_aanbieder/index.html)
+- [Verken maatregelen van aanbieder om schending auteursrechten te voorkomen](../../maatregelen/getroffen_maatregelen_van_aanbieder_voorkomen_schending_auteursrechten/index.html)
+- [Voer voorafgaand aan een aanbesteding een data beschikbaarheid, kwaliteit- en toegankelijkheidsanalayse uit.](../../maatregelen/voer_een_data_beschikbaarheid_kwaliteit_en_toegankelijkheidsanalyse_uit/index.html)
+- [Vul technische documentatie van aanbieder aan met informatie vanuit de gebruiksverantwoordelijke](../../maatregelen/vaststellen_aanleveren_informatie_technische_documentatie/index.html)
 
+
+
+
+
+
+
+
+</details>
+<details>
+<summary>Aanbevolen instrumenten</summary>
+
+
+
+
+
+
+
+
+- [Modelcontracten](../../instrumenten/modelcontractbepalingen.md)
+- Handleiding PIANIo (nog in ontwikkeling)
+
+
+
+
+
+
+
+
+</details>
+
+## Kennis delen
+
+- [Community of Practise Digitale Innovatie (CoP)](https://www.pianoo.nl/nl/themas/innovatie/netwerken/community-practice-digitale-innovaties): kennisnetwerk van PIANOo over het inkopen van digitale innovaties
+- [Werkgroep Publieke Inkoop van algoritmen en AI – Algoritmekader](https://algoritmes.pleio.nl/groups/): besloten werkgroep voor het verbeteren van de informatie over publieke inkoop op deze website
```

Comment:
Geen referentie maken naar de Werkgroep in het Algoritmekader zelf. Kennisdelen gaat niet via de Werkgroep, maar via Github/Team Algoritmekader. Ook in lijn met onderstaande.